builtin/mainmenu/tab_about.lua

```diff
@@ -27,9 +27,9 @@ local core_developers = {
 	"Krock/SmallJoker <mk939@ymail.com>",
 	"Lars Hofhansl <larsh@apache.org>",
 	"v-rob <robinsonvincent89@gmail.com>",
-	"Hugues Ross <hugues.ross@gmail.com>",
-	"Dmitry Kostenko (x2048) <codeforsmile@gmail.com>",
 	"Desour",
```

Comment:
While you're at it, could you please add my short name?
```suggestion
	"Desour/DS",
```